cli/cmd/clean.go

```diff
@@ -0,0 +1,154 @@
+package cmd
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/apex/log"
+	"github.com/spf13/cobra"
+	"github.com/tarantool/tt/cli/cmdcontext"
+	"github.com/tarantool/tt/cli/configure"
+	"github.com/tarantool/tt/cli/modules"
+	"github.com/tarantool/tt/cli/running"
+)
+
+var forceRemove bool
+
+// NewCleanCmd creates clean command.
+func NewCleanCmd() *cobra.Command {
+	var cleanCmd = &cobra.Command{
+		Use:   "clean [INSTANCE_NAME]",
+		Short: "Clean instance(s) files",
+		Run: func(cmd *cobra.Command, args []string) {
+			err := modules.RunCmd(&cmdCtx, cmd.Name(), &modulesInfo, internalCleanModule, args)
+			if err != nil {
+				log.Fatalf(err.Error())
+			}
+		},
+	}
+
+	cleanCmd.Flags().BoolVarP(&forceRemove, "force", "f", false, "do not ask for confirmation")
+
+	return cleanCmd
+}
+
+func collectFiles(list *[]string, dirname string) error {
+	err := filepath.Walk(dirname,
+		func(path string, info os.FileInfo, err error) error {
+			if err != nil {
+				return err
+			}
+
+			if !info.IsDir() {
+				*list = append(*list, path)
+			}
+			return nil
+		})
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func askConfirm(question string) (bool, error) {
+	reader := bufio.NewReader(os.Stdin)
+
+	for {
+		fmt.Printf("%s [y/n]: ", question)
+
+		resp, err := reader.ReadString('\n')
+		resp = strings.ToLower(strings.TrimSpace(resp))
+		if err != nil {
+			return false, err
+		}
+
+		if resp == "y" || resp == "yes" {
+			return true, nil
+		}
+
+		if resp == "n" || resp == "no" {
+			return false, nil
+		}
+	}
+}
+
+func clean(run *cmdcontext.RunningCtx) error {
```

Comment:
Why is there no unit test for this func?

Reply:
Added